src/mission_execution/include/mission_execution/MissionExecution.hpp

```diff
@@ -0,0 +1,36 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright 2018 Intel Corporation. All Rights Reserved.
+
+#ifndef MISSION_EXECUTION__MISSIONEXECUTION_HPP_
```

Comment:
Per Google's style guide:
https://google.github.io/styleguide/cppguide.html#The__define_Guard

Probably should rename the file to mission_execution.hpp and the define guard to MISSION_EXECUTION__MISSION_EXECUTION_HPP_